### ISSUE TITLE
Implement quanitiy discount pricing for resellers

### ISSUE THREAD

**Issue by jstrnbrg** [shiftdevices/saleor#10]

**Comment by jstrnbrg** [comment]
@elvissio please let me know if the following feature would be sufficient:
"We can globally (for all reseller/distributor identical) specify a list of quantities with the corresponding discount rate. That means the price is only conditioned to the quantity and all resellers get the same price given the same quantity input. If for some reason further discount is granted to a/some resellers, that would need to be done via a voucher"

**Comment by elvissio** [comment]
Hi, that is general idea and it's OK just as that. We can start from here as it's sufficient solution to begin with.
Forget vouchers for regular B2B sales opertaions, except in rare cases (promotions 1-2 times a year).

**Comment by elvissio** [comment]
We should be able to set conditional pricing to each customer and even better, to group of customers. I hope Saleor is(or can) supporting customer groups (or even better a tree of dependant groups).

**Comment by jstrnbrg** [comment]
@elvissio it does not currently. Is it correct this is a long-term feature which I don't need to get a quote for now?

**Comment by elvissio** [comment]
Hmm, it's true we'll have 1-3 products only, but we should have several users (100+ resellers only) from various markets. I hope this is a feature we'd need very soon.
Pricing based on qty or order value are just economic ideals which is (very) soon not enough.

**Comment by jstrnbrg** [comment]
I agree but if a reseller requests a special discount (higher than their normal discount) they would contact the sales team anyway, correct? Couldn't you then just create a 5% off voucher and it's sorted? ahah I know I keep coming back to vouchers but it's just the easiest solution for a lot of your requests. I'm confident our resellers will manage to type in a voucher code, if not it's their problem haha ;)

**Comment by elvissio** [comment]
Yes dear, forget that vouchers. In business relations you don't do that. Managing vouchers also take time which is what we want to avoid. I've tried that in B2B a lot... this is just a joke which could be used in rare promotions only. 
Summer sale, 1 week, voucher for 5% off to all: "sexonthebeach" and then forget it for a year ;)

**Comment by jstrnbrg** [comment]
ok I will update the issue so that we need individual quantity pricing for each reseller.
